### ISSUE TITLE
A wxAuiNotebook can't be docked in the center!

### ISSUE THREAD

**Issue by fcloud2020** [wxWidgets/wxWidgets#24241]
I want to dock a wxAuiNotebook control in the center of the blank space. But no matter how much I drag and drop, I can't make it happen. 

![20240122-140603](https://github.com/wxWidgets/wxWidgets/assets/70119079/04235bea-c375-4a91-aeb3-2f70920a4df4)

I looked at the source code and found that there seemed to be no judgment for docked in wxCENTER.

**Comment by vadz** [comment]
I don't think this is supported. There is just a single central pane, so it doesn't make to dock anything else in the center.

**Comment by fcloud2020** [comment]
Maybe I didn't describe this function clearly.Look like this:

![20240123-164804](https://github.com/wxWidgets/wxWidgets/assets/70119079/1027aec9-5e78-4a55-b53b-236822e3d241)

I fell this feature is very important if you have two or more monitors. 
Don't you think so?

**Comment by vadz** [comment]
Sure, having it would be better than not having it... But it looks like a relatively simple workaround could be to always keep a notebook (with a dummy page if necessary) as a center pane.

**Comment by vadz** [comment]
I've updated the title as this is not notebook-specific, no panes can be currently docked in the center.

**Comment by Omegaki113r** [comment]
Does this feature appear in any upcoming roadmaps (Docking in center)?

**Comment by vadz** [comment]
There are no precise plans, no. As I said, it would be nice to have and any PRs would be welcome.
